src/h5web/metadata-viewer/MetadataViewer.tsx

```diff
@@ -21,8 +21,8 @@ function MetadataViewer(props: Props): ReactElement {
 
   return (
     <div className={styles.metadataViewer}>
-      <EntityInfo entity={entity} />
-      <AttributesInfo attributes={entity.attributes} />
+      <EntityTable entity={entity} />
+      <AttributesTable attributes={entity.attributes} />
```

Comment:
Just renaming these two components to distinguish them from the `LinkInfo` components.